src/corner-kick/README.md

```diff
@@ -0,0 +1,57 @@
+# Corner Kick
+
+Corner Kick is an Electron app that allows for the visualization of UBC Thunderbots' simulator.
+
+## Contribute
+
+### Node
+
+You will need Node to contribute to this project. To install Node, run the following:
+
+```
+sudo apt-get install nodejs npm
```

Comment:
Just a note, looks like both `nodejs` and `npm` are available via `rosdep`, so we can bundle these dependencies into a `package.xml` file, instead of the changes made in `setup_software.sh`.